lib/membrane/core/message.ex

```diff
@@ -32,7 +32,12 @@ defmodule Membrane.Core.Message do
 
   @spec call(GenServer.server(), type_t, args_t, opts_t, timeout()) :: term()
   def call(pid, type, args \\ [], opts \\ [], timeout \\ 5000) do
-    GenServer.call(pid, message(type: type, args: args, opts: opts), timeout)
+    try do
+      GenServer.call(pid, message(type: type, args: args, opts: opts), timeout)
```

Comment:
```suggestion
      result = GenServer.call(pid, message(type: type, args: args, opts: opts), timeout)
      {:ok, result}
```

And we should handle that properly wherever it's used. We could possibly keep the old version of `call` named `call!` too.

Reply:
Currently call is not wrapping the result in the tuple `{:ok, result}`. Why do you want to change that? Also as we noticed during the last discussion on that, we are using `Message.call` only in 4 places in the whole codebase, I don't see a clear benefit of using `call!` over matching against the expected result. 
This code would end up very weird with this wrapping approach:
https://github.com/membraneframework/membrane_core/blob/master/lib/membrane/core/parent/child_life_controller/link_handler.ex#L270
The only place that requires an additional handling is here:
https://github.com/membraneframework/membrane_core/blob/master/lib/membrane/core/bin/pad_controller.ex#L180